clash-prelude/src/Clash/Class/BitPack/Internal.hs

```diff
@@ -1,102 +1,441 @@
 {-|
-Copyright  :  (C) 2019, QBayLogic B.V.
+Copyright  :  (C) 2013-2016, University of Twente,
+                  2016-2017, Myrtle Software Ltd
```

Comment:
Does this copyright notice match up with the one removed in the other file?

Reply:
Yes. This is apart of a rename:

```
BitVector -> Internal
Internal -> TH
```